client/operation_join.go

```diff
@@ -0,0 +1,14 @@
+package client
+
+import (
+	"github.com/regner/albiondata-client/log"
+)
+
+type operationJoinResponse struct {
+	Location string `mapstructure:"7"`
```

Comment:
Even if we don't need them, do we know any more of the params?

Reply:
param 2 = Character Name
param 6 = character parts json
param 26 = edition (epic-starter)